### ISSUE TITLE
Bitdefender Free started showing a threat message detected

### ISSUE THREAD

**Issue by sebadamus** [sandboxie-plus/Sandboxie#815]
**Describe the bug**
Bitdefender Free started showing a threat message detected on every program opened sandboxed.

"A potential Exploit.Generic was detected on ... SandboxieDcomLaunch.exe" (and several more originally from sandboxie dlls and programs opened by them)

**To Reproduce**
Open any sandboxed program

**Screenshots**
![imagen](https://user-images.githubusercontent.com/1787753/116346289-5665be80-a7c0-11eb-8114-67feeeabda85.png)

**System details and installed software (please provide the following information):**
 - Windows 10 Pro 20H2 compilation 19042.928
 - Sandboxie Plus 0.7.4

**Additional context**
I can recall latest windows update to be KB4023057

**Sandboxie configuration**
I have no Bitdefender rules added in sandboxie, it worked with no bitdefender error until today.
https://gist.github.com/sebadamus/248efc75dc59da202c5a9d5c180af845

**Comment by darkthepro1990** [comment]
https://www.virustotal.com/gui/file/4dce96ba033495f7ace4a211af0ed850f9f2469648297dc675a19094e98c44b4/detection

here nothing

I have bitdefender too but nothing happens
turned on bitdefender combability in sandbox?
using sandbox as portable ?
it can be false positive
depend on your actions

**Comment by sebadamus** [comment]
Thanks for you answer 👍 

> I have bitdefender too but nothing happens
> turned on bitdefender combability in sandbox?
yes as always
> using sandbox as portable ?
nope
> it can be false positive
it might be an antivirus db update?
> depend on your actions
I cant recall doing anything different :-/

This is strange :-p today I started working as usuall sandboxed, and I have no more Bitdefender warnings.  Last one was from 28/04/21.  So? it might be a Bitdefender database update or something? (I am sure I did not update windows, and changed nothing from sandboxie config.

So IT is self solved :-D (closing)
